src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -241,8 +243,9 @@ export default class FileField extends React.Component {
           </ul>
         )}
         {(maxItems === null || files.length < maxItems) &&
+          !formContext.reviewMode &&
```

Comment:
It doesn't look like there is a test for not showing the label if we aren't in review mode.  Can that be added?

Reply:
Done